src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  const selectTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
Just set the first tab as selected in case it was not found